docs/tutorials/executor_forces_tutorial.rst

```diff
@@ -295,18 +296,18 @@ Load output data from our job and return to the libEnsemble manager:
 
         return output, persis_info, calc_status
 
-Job Controller Variants
------------------------
+Executor Variants
+-----------------
 
-libEnsemble features two variants of its job controller that perform identical
+libEnsemble features two variants of its executor that perform identical
 functions, but are designed for running on different systems. For most uses,
 the MPI variant will be satisfactory. However, some systems, such as ALCF's Theta
 do not support MPI launches from compute nodes. On these systems libEnsemble is
-run either on launch nodes or using a proxy launch mechanism to submit
-jobs from compute nodes. One such mechanism is a scheduling utility called
-Balsam_ which runs on a separate node. The Balsam Job Controller variant interacts
+run either on launch nodes or uses a proxy launch mechanism to submit
+sub-jobs from compute nodes. One such mechanism is a scheduling utility called
```

Comment:
Why "sub-jobs" here?

Reply:
libEnsemble submits "tasks" to Balsam, which are launched as "jobs" since they're enqueued and (I believe) launched to separate allocations. This is definitely confusing and worth discussing in a meeting.

Reply:
I think it should be tasks.

Reply:
I think it's tasks now. Does this "Executor Variants" paragraph still make sense as a whole?